### PR TITLE
feat: Redirect to their respective profiles for all the team

### PR DESCRIPTION
**Pull request by Panquesito7**
#### Description of the change

- Redirect to their respective profiles for all the team.
  - UNTESTED, might not work.

#### Checklist

**I agree to the following :-**

*   Added description of the change
*   I've read the [contributing guidelines](https://github.com/web3community/devprotocol.xyz/blob/main/CONTRIBUTING.md)
*   Search previous suggestions before making a new PR, as yours may be a duplicate.
*   I acknowledge that all my contributions will be made under the project's license.

Notes (**optional**): 

<a href="https://gitpod.io/#https://github.com/web3community/devprotocol.xyz/pull/149"></a>

### CODE REVIEW

**Review by vinzvinci — COMMENTED** [review]
I also want the link in their name, what do u think?

**Comment by Panquesito7** [comment]
> I also want the link in their name, what do u think?

Yep, just added. 😄

File: src/components/Developers/Team.astro [hunk 1]
```diff
@@ -1,74 +1,88 @@
...
-        img: '/assets/teams/mariko.png'
+        img: '/assets/teams/mariko.png',
+        link: "https://www.linkedin.com/in/mariko-miyamoto-96a907112/"
     },
     {
         name: 'Masako Arita',
         title: 'General Manager',
-        img: '/assets/teams/masako.jpeg'
+        img: '/assets/teams/masako.jpeg',
+        link: "https://medium.com/@masako_76810"
```
Comment (vinzvinci) [inline src/components/Developers/Team.astro:25]: https://medium.com/@masako_76810/about - I think better

File: src/components/Developers/Team.astro [hunk 2]
```diff
@@ -1,74 +1,88 @@
...
-        img: 'https://github.com/pdotall.png'
+        img: 'https://github.com/pdotall.png',
+        link: "https://twitter.com/pdotal"
     },
     {
         name: 'Andrew Giunta',
         title: 'Ecosystem Growth',
-        img: '/assets/teams/soon.png'
+        img: '/assets/teams/soon.png',
+        link: "https://discordapp.com/users/234895291868905472/"
```
Comment (vinzvinci) [inline src/components/Developers/Team.astro:61]: I think don't add this. He got no real profile in discord.

Comment (shuklaritvik06) [reply src/components/Developers/Team.astro:61]: Yep

**Comment by shuklaritvik06** [comment]
And for masako do we have any other profile than medium ? @vinzvinci

File: src/components/Developers/Team.astro (hunk 2 above)
Comment (Panquesito7) [reply src/components/Developers/Team.astro:61]: > I think don't add this. He got no real profile in discord.

What link should we add then? If we don't add a link, it won't work properly.

Comment (Panquesito7) [reply src/components/Developers/Team.astro:61]: Also when I click on that it *does* lead me to their Discord profile.

image

Comment (vinzvinci) [reply src/components/Developers/Team.astro:61]: there's coming soon added in the team page

Comment (shuklaritvik06) [reply src/components/Developers/Team.astro:61]: > > I think don't add this. He got no real profile in discord.
> 
> What link should we add then? If we don't add a link, it won't work properly.

Yes it lead to their discord prof but I think vinz is saying that he don't have real name on discord so maybe you can try to redirect to the same page when anyone click?

Comment (Panquesito7) [reply src/components/Developers/Team.astro:61]: What should we add for now?

Comment (vinzvinci) [reply src/components/Developers/Team.astro:61]: None.

Comment (Panquesito7) [reply src/components/Developers/Team.astro:61]: > None.

If we don't add a link, it won't work properly (there will be an error).

Comment (vinzvinci) [reply src/components/Developers/Team.astro:61]: U can add https://devprotocol.xyz/developers

**Review by vinzvinci — APPROVED** [review]

**Review by krishguptadev — COMMENTED** [review]
LGTM!

**Pull request merged**